components/molecules/Form/Form.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {Formik, Form as FormikForm} from 'formik'
+import styles from './Form.module.css'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+
+// TODO Update form title with dynamic heading tag.
+
+export default function Form({
+  className,
+  children,
+  id,
+  formDefaults,
+  title,
+  validationSchema
+}) {
+  return (
+    <Formik
+      initialValues={formDefaults}
+      validationSchema={validationSchema}
+      onSubmit={(values, {setSubmitting}) => {
+        setTimeout(() => {
+          alert(JSON.stringify(values, null, 2))
+          setSubmitting(false)
+        }, 400)
+      }}
+    >
+      <FormikForm id={id} className={cn(styles.form, className)}>
+        {title && <h1 className={styles.title}>{title}</h1>}
+        {children}
+        <button type="submit">Submit</button>
```

Comment:
Should we try to pull the button text from GF as well (probably a later TODO)?

Reply:
I think that is a good idea as a later TODO. I believe there are a few GF form settings that we're missing which would make it a good separate PR. I'll get the directory name updated now.